components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -415,18 +416,52 @@ def initially_open_owner(self):
 def paths_to_object(conn, experimenter_id=None, project_id=None,
                     dataset_id=None, image_id=None, screen_id=None,
                     plate_id=None, acquisition_id=None, well_id=None,
-                    group_id=None):
+                    group_id=None, page_size=settings.PAGE):
```

Comment:
While `settings.PAGE` is unlikely to be mutated during execution, in Python keyword argument defaults are only evaluated once. If for some reason there's some funky import order, etc. you have the possibility of not getting what you expect here. I would suggest a documented, immutable default value and then the setting of the default within the function if possible.